src/lib/staged_ledger/staged_ledger.ml

```diff
@@ -2202,7 +2202,19 @@ module T = struct
             in
             [%log internal] "Generate_staged_ledger_diff" ;
             let diff, log =
-              O1trace.sync_thread "generate_staged_ledger_diff" (fun () ->
+              O1trace.sync_thread "generate_staged_ledger_diff"
+                (fun
+                  ()
+                  :
```

Comment:
TIL about this syntax. This is much nicer than what I've been doing.

Reply:
@mrmr1993 pun intended :D ?

This can probably be removed, I think I needed this only for debugging the compiler errors at some point